### DISCUSSION TITLE
Fleetbase Open Source Going Live!

### DISCUSSION THREAD

**Discussion by shivthakker** [fleetbase/fleetbase-js#5] [Show and tell]
Looking forward to releasing Fleetbase open source for you all

**Comment by rokello1** [comment]
Thanks for this. I am trying this today but I am getting an error on the yml file as below:
can you help,

WARNING: The GITHUB_AUTH_KEY variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services: 'httpd'

**Reply by roncodes** [reply]
Try upgrading your docker

**Reply by rokello1** [reply]
I am running the latest docker version
Docker version 26.1.3,
But i am still getting the same error from the compose yml file.
